alica_engine/include/engine/collections/RobotEngineData.h

```diff
@@ -37,7 +37,7 @@ class RobotEngineData
      */
     int64_t makeUniqueId(const std::string& s) const;
 
-    const AlicaEngine* _engine;
+    AlicaEngine* _engine;
```

Comment:
This is really a weird non-const back pointer

Reply:
To be fixed later.